core/primitives/src/version.rs

```diff
@@ -17,6 +17,6 @@ pub const DB_VERSION: DbVersion = 1;
 pub type ProtocolVersion = u32;
 
 /// Current latest version of the protocol.
-pub const PROTOCOL_VERSION: ProtocolVersion = 23;
+pub const PROTOCOL_VERSION: ProtocolVersion = 24;
```

Comment:
We should stop this workaround and only change protocol version when there is actually a protocol change. We already store the binary version in store so it should be possible to do storage migration.

Reply:
Cool, can you give an example of using storage migration?

Reply:
Looks this is the first storage change after binary version in store so there seems not example :) Also do we have the "check storage version, run migrate" logic in neard? If not there should be three adds in our infra:
- add a pytest storage_migration.py or extend state_migration.py to perform and test
- in deploy, perform storage_migrations when there's storage_migration but no state_migration
- in nearup, when downloaded binary has new version, also download proper storage migration script to perform migration instead of nuke data

Reply:
@bowenwang1996 ptal
migration infra is available in neard and it's very convenient to use